app-frontend/src/app/components/pages/project/layer/aoi/index.js

```diff
@@ -86,15 +86,7 @@ class ProjectLayerAoiController {
             .then(paginatedResponse => {
                 fetchedList = paginatedResponse.results;
                 this.itemList = fetchedList
-                    .filter(fl => fl.geometry && fl.id !== this.layer.id).map(fl => {
-                        return {
-                            id: fl.id,
-                            name: fl.name,
-                            date: fl.createdAt,
-                            colorGroupHex: fl.colorGroupHex,
-                            geometry: fl.geometry
-                        };
-                    });
+                    .filter(fl => fl.geometry && fl.id !== this.layer.id);
```

Comment:
Removed unnecessary item field mapping to ensure all fields are present. Instead, missing fields for vectors are handled at the template level.